### PR TITLE
refact(logging): make trace wrapper part of public API

### PR DESCRIPTION
**Pull request by antmat**
I need this to use the same wrapper in loggingv2 service

### CODE REVIEW

**Comment by 3Hren** [comment]
LGTM

File: src/context.cpp [hunk 1]
```diff
@@ -43,7 +44,7 @@
 
 #include <deque>
 #include <boost/algorithm/string/join.hpp>
-#include <cocaine/detail/trace/logger.hpp>
+
```
Comment (3Hren) [inline src/context.cpp]: No extra line please.

**Comment by antmat** [comment]
updated

**Comment by antmat** [comment]
How does it come that you being a python programmer don't like 2 blank lines?)

**Pull request merged**